### DISCUSSION TITLE
As secretary, I want to be able to approve bills that have made its way through the workflow

### DISCUSSION THREAD

**Discussion by michnotmeesh** [a-doe-bo/hi-doe-bill-tracker#33] [General]
As secretary, I want to be able to approve bills that have made its way through the workflow

1. Log in
2. Navigate to "Final Testimonies"
3. Click on bills
4. If secretary approves, secretary can use "Add Testimony to bill" button and email all partners interested in the bill.

**Comment by Cole-House** [comment]
Do Secretaries have the ability to approve bills?

**Reply by michnotmeesh** [reply]
No. Secretary is the mailman.
